viz/cmd/tap.go

```diff
@@ -299,19 +313,37 @@ func requestTapByResourceFromAPI(ctx context.Context, w io.Writer, k8sAPI *k8s.K
 }
 
 func writeTapEventsToBuffer(w io.Writer, tapByteStream *bufio.Reader, options *tapOptions) error {
-	switch options.output {
-	case "":
+	output := options.output
+
+	switch {
```

Comment:
we can rewrite this as 
`
switch options.output{
`
This will avoid the `==` checks in the switch case and make it cleaner

Reply:
Yes!  But with `switch options.output{` it requires string for the comparison in the switch. so we will not be able to check  `strings.HasPrefix(output, jsonPathOutput)` with this approach